src/components/Request/Pay/Views/GeneralViews/AlreadyPaid.view.tsx

```diff
@@ -7,22 +7,45 @@ import * as utils from '@/utils'
 import Link from 'next/link'
 
 export const AlreadyPaidLinkView = ({ requestLinkData }: { requestLinkData: _consts.IRequestLinkData | undefined }) => {
-
-    const chainName = consts.supportedPeanutChains && consts.supportedPeanutChains.find((chain) => chain.chainId == requestLinkData?.chainId)?.name
+    const chainName =
+        consts.supportedPeanutChains &&
+        consts.supportedPeanutChains.find((chain) => chain.chainId == requestLinkData?.chainId)?.name
     const tokenSymbolAvailable: boolean = !!requestLinkData?.tokenSymbol
     const tokenAmountAvailable: boolean = !!requestLinkData?.tokenAmount
     const chainAvailable: boolean = !!requestLinkData?.chainId
     const recipientAddressAvailable: boolean = !!requestLinkData?.recipientAddress
-    const dataAvailable: boolean = tokenSymbolAvailable || tokenAmountAvailable || chainAvailable || recipientAddressAvailable
-    
+    const dataAvailable: boolean =
+        tokenSymbolAvailable || tokenAmountAvailable || chainAvailable || recipientAddressAvailable
+
     return (
         <div className="flex w-full flex-col items-center justify-center gap-6 py-2 pb-20 text-center">
+            {(requestLinkData?.reference || requestLinkData?.attachmentUrl) && (
```

Comment:
>  {(requestLinkData?.reference || requestLinkData?.attachmentUrl) && (
Comment

Q: Why do this? is it necessary? We already check for existence of both of  these individually later